components/SuccessSection/index.js

```diff
@@ -0,0 +1,56 @@
+import { useRouter } from "next/router";
+import { StyledButton } from "../../styles/StyledButton";
+import { StyledRoundCounter } from "../../styles/StyledRoundCounter";
+import { useEffect, useState } from "react";
+import useAudio from "../useAudio";
+import useStore from "../useStore";
+import { StyledInstruction } from "../../styles/StyledInstruction";
+import { StyledQuote } from "../../styles/StyledQuote";
+
+export default function SuccessSection({ breathIntervalDelay }) {
+  const resetIceBathCountdown = useStore(
+    (state) => state.resetIceBathCountdown
+  );
+  const switchIceBathCountdownIsActive = useStore(
+    (state) => state.switchIceBathCountdownIsActive
+  );
+  const iceBathCountdownIntervalId = useStore(
+    (state) => state.iceBathCountdownIntervalId
+  );
+  const resetIceBathCount = useStore((state) => state.resetIceBathCount);
+  const switchIceBathCountIsActive = useStore(
+    (state) => state.switchIceBathCountIsActive
+  );
+  const iceBathIntervalId = useStore((state) => state.iceBathIntervalId);
+
```

Comment:
This is not necessary, but we could switch the following

``` js 
const resetIceBathCountdown = useStore(
(state) => state.resetIceBathCountdown
);
const switchIceBathCountdownIsActive = useStore(
(state) => state.switchIceBathCountdownIsActive
);
const iceBathCountdownIntervalId = useStore(
(state) => state.iceBathCountdownIntervalId
);
const resetIceBathCount = useStore((state) => state.resetIceBathCount);
const switchIceBathCountIsActive = useStore(
(state) => state.switchIceBathCountIsActive
);
const iceBathIntervalId = useStore((state) => state.iceBathIntervalId);
```

to be 

``` js 
const {
  resetIceBathCountdown,
  switchIceBathCountdownIsActive,
  iceBathCountdownIntervalId,
  resetIceBathCount,
  switchIceBathCountIsActive,
  iceBathIntervalId,
} = useStore((state) => state);
```

Reply:
Thanks for the suggestion. We were strictly told not to destructure zustand stores that way as it might cause many errors (rendering etc.) so I would prefer keeping it the way it is, as it is best practice :)

Reply:
I have spent more time looking at the zustand docs and i think you were given good advice here. Apologies for suggesting something otherwise. 

My follow up would be are these needed at all in this component? For example, i dont see `resetIceBathCountdown ` being used anywhere.